assets/onboarding/features/feature-description.jsx

```diff
@@ -31,9 +26,9 @@ const FeatureDescription = ( {
 			</>
 		) }
 
-		{ errorMessage && (
+		{ error && (
 			<p className="sensei-onboarding__error-message">
-				{ errorMessage }
+				{ error }
 				{ onFeatureRetry && (
```

Comment:
The error & retry block here could be moved out of `<FeatureDescription />`, as this is specific to the install feedback and not used elsewhere.

Reply:
Updated here: https://github.com/Automattic/sensei/pull/3190/commits/c4865399a4d99a15e1f389c62ca2d9321a98cadf